pkg/data/dashboard/namespace.go

```diff
@@ -11,14 +12,26 @@ import (
 )
 
 func addCattleGlobalNamespaces(ctx context.Context, k8s kubernetes.Interface) error {
+	if features.Fleet.Enabled() {
+		_, err := k8s.CoreV1().Namespaces().Get(ctx, "fleet-local", metav1.GetOptions{})
+		if apierrors.IsNotFound(err) {
+			_, err = k8s.CoreV1().Namespaces().Create(ctx, &corev1.Namespace{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "fleet-local",
+				},
+			}, metav1.CreateOptions{})
+		}
+		if err != nil {
```

Comment:
this needs to be `else if`

Reply:
No, because the err is reassigned in the body.  If it's `else if` the err in the body of the iff won't be handled.

Reply:
Ah I see, it is the same as here: https://github.com/rancher/rancher/pull/34798/files#r712303455